src/geometries/BoxGeometry.js

```diff
@@ -95,32 +95,32 @@ class BoxBufferGeometry extends BufferGeometry {
 
 		function buildPlane( u, v, w, udir, vdir, width, height, depth, gridX, gridY, materialIndex ) {
 
-			var segmentWidth = width / gridX;
-			var segmentHeight = height / gridY;
+			const segmentWidth = width / gridX;
+			const segmentHeight = height / gridY;
 
-			var widthHalf = width / 2;
-			var heightHalf = height / 2;
-			var depthHalf = depth / 2;
+			const widthHalf = width / 2;
+			const heightHalf = height / 2;
+			const depthHalf = depth / 2;
 
-			var gridX1 = gridX + 1;
-			var gridY1 = gridY + 1;
+			const gridX1 = gridX + 1;
+			const gridY1 = gridY + 1;
 
-			var vertexCounter = 0;
-			var groupCount = 0;
+			let vertexCounter = 0;
+			let groupCount = 0;
 
-			var ix, iy;
+			let ix, iy;
```

Comment:
In such cases, I would not just turn `var` into `let`. Since `let` is a true block-scope variable, it's just natural to declare `ix` and `iy` in the `for` loops.

Reply:
You mean removing that line and declaring then in the loop, right?

```js
for ( let iy = 0; iy < gridY1; iy ++ ) {
```

```js
for ( let ix = 0; ix < gridX1; ix ++ ) {
```

Reply:
Exactly! This is the way I would implement it now from scratch.

Reply:
Will merge and fix.

Reply:
can implement it that way in future PRs. Was just doing one thing at a time initially